src/components/StyledForm.js

```diff
@@ -0,0 +1,49 @@
+import {useState} from 'react';
+import styled from 'styled-components';
+
+import useStore from '../hooks/useStore';
+
+export default function StyledForm() {
+	const [inputValue, setInputValue] = useState('');
```

Comment:
Please have a look at my comment below regarding the `onChange` function. I think you won't need this state 🙂